src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -88,7 +90,18 @@ private void ShouldAllowNavigation(object sender, NavigatingCancelEventArgs e)
         /// <param name="link"></param>
         public void NavigateToPage(Uri link)
         {
-            this.documentationBrowser.NavigateToString(this.viewModel.GetContent());
+            Dispatcher.BeginInvoke(new Action(() =>
```

Comment:
I ran into thread issues with this. My guess is that because im calling the `NavigateToPage` from the viewmodel with the node annotation stuff its being called on the main thread and not the UI thread.

With the error messages we are calling it from the `InfoBubble` view which is still on the UI thread and therefor we didnt have any issues.

The dispatcher is still a bit of a mystery to me sometimes, so might be totally wrong here. This was the only way i could make it work.